lib/environment.libsonnet

```
@@ -0,0 +1,84 @@
+{
+  /**
+  Creates an environment variable with a static value.
+  Parameters:
+    - name: string - The name of the environment variable.
+    - value: string - The value to assign to the environment variable.
+  */
+  envVariable(name, value): {
+    spec+: {
+      env+: [
+        {
+          name: name,
+          value: value,
+        },
+      ],
+    },
+  },
+  /**
+  Creates an environment variable whose value is sourced from a secret.
+  Parameters:
+    - name: string - The name of the environment variable.
+    - secretRef: string - The name of the secret resource. The key used in the secret is the same as the environment variable name.
+  */
+  envVariableSecret(name, secretRef): {
```

Dette føler jeg er en mer ergonomisk måte å gjøre det på, med et optional felt som defaulter til det samme som `name`). Kan også gjøres på job-varianten.
```suggestion
  envVariableSecret(name, secretRef, key=name): {
```